Battery.elm

```diff
@@ -1,35 +1,48 @@
 module Battery exposing (Battery, recommend)
 
-import Usage exposing (Usage)
+import List exposing (head, map, repeat, sortBy, sum)
 import Load exposing (getAmps)
-import List exposing (sum, map, repeat, sortBy, head)
+import Usage exposing (Usage)
+
+
 -- MODEL
 
+
 type alias Battery =
     { name : String
     , maxDischargePercentage : Int
     , ah : Int
     , price : Float
     }
 
-liFePO4 = Battery "LiFePO4" 80
 
 batteries : List Battery
 batteries =
+    let
+        liFePO4 =
+            Battery "LiFePO4" 80
```

Comment:
just wanted to have it private within this scope

Reply:
:+1: